Client/mods/deathmatch/logic/CClientPed.h

```diff
@@ -573,6 +573,7 @@ class CClientPed : public CClientStreamElement, public CAntiCheatModule
     CStatsData*                              m_stats;
     CControllerState*                        m_currentControllerState;
     CControllerState*                        m_lastControllerState;
+    CControllerState                         m_rawControllerState;
```

Comment:
A future programmer might be confused about the difference between `m_currentControllerState` and `m_rawControllerState` (specifically, what `current` has that `raw` doesn't have). I would suggest you include comments here that explain this.